src/line_info.c

```diff
@@ -134,6 +134,96 @@ static struct line_info *line_info_ret_queue(struct history *hst)
     return line;
 }
 
+/* Toggles the peer_list enabled variable for every line in hsitory. This allows us to
+ * always have lines word wrapped at the edge of the visible window
+ */
+void line_info_peerlist_toggle(struct history *hst, bool enabled)
+{
+    struct line_info *line = hst->line_root;
+
+    while (line) {
+        line->show_peerlist = enabled;
+        line = line->next;
+    }
+}
+
+/* Prints `n` chars from `s` to `win`.
+ *
+ * Return true if the string contains a newline or tab byte.
+ */
+static bool print_n_chars(WINDOW *win, const char *s, size_t n)
+{
+    bool newline = false;
+    char ch;
+
+    for (size_t i = 0; i < n && (ch = s[i]); ++i) {
+        wprintw(win, "%c", ch);
+
+        if (ch == '\n' || ch == '\t') {
```

Comment:
Should tab be treated as a newline? Not a space character?

Reply:
Yes, tabs were actually parsed correctly to begin with, not sure why I did that.